styles.css

```diff
@@ -352,4 +351,112 @@
   .btn:disabled {
     background-color: #2a2b33;
   }
+
+  .contact-form .flex-container-column,
+  .flex .flex-container-column {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .contact-form {
+    margin-top: 75px;
+  }
+
+  .mobile-form {
+    padding: 20px;
+  }
+
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 40px;
+    gap: 24px;
+  }
+
+  .contact-form div h2 {
+    margin-bottom: 20px;
+    font-size: 32px;
+    line-height: 44px;
+    color: #9390ff;
+    font-weight: 500;
+    font-family: 'Roboto', sans-serif;
+    text-align: center;
+  }
+
+  .contact-form div p {
+    font-size: 16px;
+    line-height: 24px;
+    color: #f5f5ff;
+    font-family: 'Roboto', sans-serif;
```

Comment:
- [ ] Your style of this text matches almost perfectly with the Figma template, but the font-family of the template is different from 'Roboto' for this piece of text. Try changing the font-family to match the Template.